pages/workflow/plan.mdx

```diff
@@ -0,0 +1,79 @@
+---
+layout: 'guide'
+title: '1. Plan analytics update in Avo branch'
+abstract: ''
+---
+
+import Link from '../../components/Link';
+
+## 1.1 Decide what to track
+
+_Role: Product or Data (This initiative is usually led by a product oriented person)_
+
+The hardest part in the whole analytics release process is usually to decide what events and properties you need, to be able to measure the success of the product change. There are many paths to a decision, but no matter which one you take, we recommend doing it as early as possible in the development process.
+
+We always recommend starting by formulating the metrics you hope to move with your change. Our favorite way to do this is what we like to call a <Link passHref href="/blog/tracking-the-right-product-metrics"><a>“Purpose Meeting”</a></Link>. The outcomes of that meeting are:
+1. Goals – what does success look like?
+2. Metrics – how will we measure how successful we are?
+3. Data – what analytics events do we need for our success metrics?
+
+The result of the purpose meeting is usually a draft of events that someone takes forward to the next step.
+
+> [Learn what's the “Purpose Meeting”](https://www.avo.app/blog/tracking-the-right-product-metrics).
+
+## 1.2 Create a branch in Avo
+
+_Role: Product or Data. (This is also sometimes done by engineering, particularly when the engineering team is product driven.)_
+
+The first thing you do in Avo, for a new feature or iteration, is to create a new branch.
+
+A branch allows you to draft changes to the tracking plan in isolation, without impacting the main branch, which contains the single source of truth of what’s currently being tracked.
+
+To create a new branch, first make sure you are on the main branch. Head to the <Link passHref href="/workspace/branches#branches-overview"><a>Branches overview</a></Link> screen by clicking the Avo logo, and from there click “Create branch”.
```

Comment:
This doc does not match the video below. We should make sure they are consistent. Should we update the text here to match what is happening in the video @JuliaSifOlafs ?